crmc.sh

```diff
@@ -0,0 +1,41 @@
+package: CRMC
+version: v1.5.4
+requires:
+  - boost
+  - HepMC
+---
+#!/bin/bash -ex
+
+ARCHIVE="crmc-${PKGVERSION#v}-src.tgz"
+URL="http://service-spi.web.cern.ch/service-spi/external/MCGenerators/distribution/crmc/${ARCHIVE}"
+
+cd $SOURCEDIR
+curl -O "${URL}"
+tar xz --strip-components 2 -f ${ARCHIVE}
+
+cd $BUILDDIR
+
+cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT
+
+make ${JOBS+-j $JOBS} all
+make install
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
+MODULEFILE="$MODULEDIR/$PKGNAME"
+mkdir -p "$MODULEDIR"
+cat > "$MODULEFILE" <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0 yaml-cpp/$YAML_CPP_VERSION-$YAML_CPP_REVISION boost/$BOOST_VERSION-$BOOST_REVISION autotools/$AUTOTOOLS_VERSION-$AUTOTOOLS_REVISION
+# Our environment
+setenv LHAPDF_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
```

Comment:
Watch out, here it should be `CRMC_ROOT` instead of `LHAPDF_ROOT`, could you fix it?